Do not send status update with STOPPING=1 during reload. Otherwise,
systemd may transition from reload to stop-sigterm because of this.

Index: unbound-1.6.7/daemon/daemon.c
===================================================================
--- unbound-1.6.7.orig/daemon/daemon.c
+++ unbound-1.6.7/daemon/daemon.c
@@ -650,7 +650,10 @@ daemon_fork(struct daemon* daemon)
 	log_info("start of service (%s).", PACKAGE_STRING);
 	worker_work(daemon->workers[0]);
 #ifdef HAVE_SYSTEMD
-	sd_notify(0, "STOPPING=1");
+	if (daemon->workers[0]->need_to_exit)
+		sd_notify(0, "STOPPING=1");
+	else
+		sd_notify(0, "RELOADING=1");
 #endif
 	log_info("service stopped (%s).", PACKAGE_STRING);
 
